### DISCUSSION TITLE
UX improvement in selecting date range for reporting

### DISCUSSION THREAD

**Discussion by seedor-io** [btcpayserver/btcpayserver#5921] [Ideas / Feature Requests]
It's a miner improvement in my opinion, but I'd like to select both start and end dates in one form instead of having to click on a new field for the end date. Attached a brief video of how for instance SkyScanner and many others have implemented this: 
![lv_0_20240410075124](https://github.com/btcpayserver/btcpayserver/assets/122058767/23f511eb-c147-498a-94aa-45492229d76f)

**Comment by dennisreimann** [comment]
Note for implementation: We're using the Flatpickr library and using their [range select](https://flatpickr.js.org/examples/#range-calendar) we could support this.

**Reply by pavlenex** [reply]
Do we have this used anywhere other than on reports? Would be good to tackle globally. I remember we have date-pickers in payment requests and perhaps POS.

**Reply by dennisreimann** [reply]
To my knowledge a date range is only used on the crowdfund edit page as well, thought it might not work there, because only start or end are required. Would need to check the code.
